jumpscale/packages/vdc_dashboard/bottle/controller.py

```diff
@@ -0,0 +1,275 @@
+from beaker.middleware import SessionMiddleware
+from bottle import Bottle, request, HTTPResponse, abort
+import random
+
+from jumpscale.loader import j
+from jumpscale.packages.auth.bottle.auth import SESSION_OPTS, controller_authorized
+from jumpscale.packages.vdc_dashboard.bottle.vdc_helpers import get_vdc, threebot_vdc_helper
+from jumpscale.sals.vdc.size import VDC_SIZE
+
+
+app = Bottle()
+
+
+def _get_vdc_dict():
+    vdc = get_vdc()
```

Comment:
do we need to vdc.load info() ?
I think kubernetes and 3bot data isn't filled at the begining

Reply:
get_vdc() itself passes `load_info=True`